app/controllers/articles_controller.rb

```diff
@@ -1,10 +1,12 @@
 class ArticlesController < ApplicationController
+  before_action :set_article, only: [:show, :edit, :update]
+
   def new
     @article = Article.new
```

Comment:
**todo:** Scope this call.

You have scoped all your calls to ensure they belong to the current user, but you must change this one too. You want to do it here to ensure consistency across the controller. It will also help reduce the likelihood of any strange bugs in the future.